package.json

```diff
@@ -1,34 +1,50 @@
 {
-  "name": "unmock",
-  "displayName": "unmock-js",
-  "version": "0.0.40",
-  "main": "dist/index.js",
-  "bin": {
-    "unmock": "./dist/cli.js"
-  },
   "license": "MIT",
   "scripts": {
-    "prepare": "tsc --outDir dist",
-    "pretest": "yarn prepare",
-    "test": "jest"
+    "clean": "rm -rf node_modules ./packages/*/dist ./packages/*/tsconfig.tsbuildinfo",
+    "compile": "tsc --build tsconfig.build.json",
+    "compile:clean": "tsc --build tsconfig.build.json --clean",
+    "watch": "tsc --build tsconfig.build.json --watch",
+    "release": "npm run clean && npm install && npm ci && lerna publish --exact --force-publish=unmock --include-merged-tags",
+    "postinstall": "lerna run prepare && npm run compile",
+    "lint": "prettier-check '**/*.{js,ts}'",
+    "lint-fix": "prettier '**/*.{js,ts}' --write",
+    "test": "jest --verbose",
+    "test:clean": "jest --clearCache",
+    "test:watch": "jest --verbose --watchAll",
+    "testonly": "npm test",
```

Comment:
I thought everything in `scripts` is run with `npx`? Is it refering to the `test` defined above? How does this work?

Reply:
it needs to be this way (see `apollo-server`). if you remove the `npm` it doesn't work.